docs/specs/tsk-ha5iau/2026-08-28-lora-off-grid-transport-design.md

```
@@ -240,24 +261,77 @@ other; the prototype defines exactly one config and both modules load it.
+#### Duty-Cycle Budget (EU_868, LongFast)
+
+On-air time of one frame at the pinned preset (SF11, 250 kHz, coding rate 4/5, Meshtastic
+16-symbol preamble). Symbol time `T_sym = 2^SF / BW` = 8.192 ms; DE = 0 because T_sym
+= 8.192 ms < 16 ms so low-data-rate optimisation is not mandatory. Total on-air time:
+`ToA = (n_preamble + 4.25) * T_sym + n_payload * T_sym` (Semtech AN1200.13 form, see
+References).
+
+For the payload-symbol count, with header IH = 0 (explicit header), CRC on (CRC = 1),
+coding rate CR = 1 (4/5), DE = 0, SF = 11:
+
+    n_payload = 8 + max(ceil((8*PL - 4*SF + 28 + 16*CRC - 20*IH) /
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)